### PR TITLE
doc: add collaborations information

### PR DESCRIPTION
**Pull request by huoqifeng**
Fixes: #352

Signed-off-by: huoqifeng <huoqif@cn.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,3 +40,12 @@ Please refer to the instructions mentioned in the following [doc](./docs/addnewp
 ## Contribution
 
 This project uses [the Apache 2.0 license](./LICENSE). Contribution to this project requires the [DCO 1.1](./DCO1.1.txt) process to be followed.
+
+## Collaborations
+
+Slack: [#confidential-containers-peerpod](https://cloud-native.slack.com/archives/C04A2EJ70BX)  
+Zoom meeting: https://zoom.us/j/97726916398 on each `Wednesday` at.
+- 9:00  -- 10:00 GMT Winter time
```
Comment (stevenhorsman) [inline README.md:48]:
```suggestion
-  8:00 - 9:00 UTC
```

File: README.md [hunk 2]
```diff
@@ -40,3 +40,12 @@ Please refer to the instructions mentioned in the following [doc](./docs/addnewp
 ## Contribution
 
 This project uses [the Apache 2.0 license](./LICENSE). Contribution to this project requires the [DCO 1.1](./DCO1.1.txt) process to be followed.
+
+## Collaborations
+
+Slack: [#confidential-containers-peerpod](https://cloud-native.slack.com/archives/C04A2EJ70BX)  
+Zoom meeting: https://zoom.us/j/97726916398 on each `Wednesday` at.
```
[the remark this answers is not in the document]
Comment (huoqifeng) [reply README.md:47]: @stevenhorsman thanks! @bpradipt any comments here?

File: README.md (hunk 1 above)
Comment (huoqifeng) [reply README.md]: Thanks @knrt10 and @stevenhorsman , fixed.

File: README.md [hunk 3]
```diff
@@ -40,3 +40,9 @@ Please refer to the instructions mentioned in the following [doc](./docs/addnewp
 ## Contribution
 
 This project uses [the Apache 2.0 license](./LICENSE). Contribution to this project requires the [DCO 1.1](./DCO1.1.txt) process to be followed.
+
+## Collaborations
+
+Slack: [#confidential-containers-peerpod](https://cloud-native.slack.com/archives/C04A2EJ70BX)
```
Comment (liudalibj) [inline README.md:46]: should we add something like:

> If you need to join the workspace, you should be able to follow https://communityinviter.com/apps/cloud-native/cncf

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (stevenhorsman) [reply README.md]: Ariel has created the meeting, so the link is https://zoom.us/j/94601737867?pwd=MEF5NkN5ZkRDcUtCV09SQllMWWtzUT09

[the remark this answers is not in the document]
Comment (huoqifeng) [reply README.md]: Thanks @stevenhorsman , I guess it's https://zoom.us/j/94601737867 without `?pwd=MEF5NkN5ZkRDcUtCV09SQllMWWtzUT09`?

[the remark this answers is not in the document]
Comment (huoqifeng) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (huoqifeng) [reply README.md]: Thanks @liudalibj , fixed

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (stevenhorsman) [reply README.md]: Yeah, I'm not sure what the plaintext password is, so easier to bundle it with the link!

**Review by stevenhorsman — APPROVED** [review]
LGTM - thanks QiFeng!

**Review by liudalibj — APPROVED** [review]
LGTM

**Pull request merged**